tools/src/test/resources/adl2-tests/validity/structure/openEHR-TEST_PKG-EVALUATION.VUNP_use_node_points_to_use_node.v1.0.0.adls

```diff
@@ -0,0 +1,70 @@
+archetype (adl_version=2.0.7; rm_release=1.0.2)
```

Comment:
Is this archetype used anywhere?

Reply:
Everything in the validity directory is automatically used by the test. If it contains regression = <"SOME_ERROR">, the test asserts that that error is present. If "PASS", it must validate. If you remove the code changes, you will see this archetype causing a failed test.